src/main/resources/liquibase.properties

```diff
@@ -0,0 +1,5 @@
+changeLogFile=src/main/resources/db/db-changelog.yaml
```

Comment:
What about test data? Will we add another step for that or do we want to approach it  differently?

Reply:
on preview environments there will be leftovers from functional tests, so any data will be from there if needed.